libs/base/ui/structure/collapsible/src/collapsible.component.ts

```diff
@@ -44,6 +55,16 @@ export class CollapsibleComponent
 
   protected onToggle(): void {
     this.open = this.details?.open;
+    if (this.isManuallyOpened && this.open) {
+      if (featureVersion >= '1.2') {
```

Comment:
Maybe it make sense to check version first? `isManuallyOpened` doesn't even exist in earlier versions.
```suggestion
    if (featureVersion >= '1.2') {
      if (this.isManuallyOpened && this.open) {
```